yarn.lock

```diff
@@ -0,0 +1,11602 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Looks like this repo is using `package-lock.json` (not `yarn.lock`), mind deleting this and updating the package-lock.json?